test/controllers/PaypalControllerSpec.scala

```diff
@@ -80,22 +83,24 @@ class PaypalControllerFixture(implicit ec: ExecutionContext) extends MockitoSuga
   val controller: PaypalController = new PaypalController(mockPaymentServices, mockCorsConfig, supportConfig, mockCsrfCheck, mockCloudwatchMetrics, mockOphanService)
 
   def numberOfCallsToStoreMetaDataMustBe(times: Int): Unit = {
-    def captor[A <: AnyRef](implicit classTag: ClassTag[A]): A =
-      ArgumentCaptor.forClass[A](classTag.runtimeClass.asInstanceOf[Class[A]]).capture()
-
-    Mockito.verify(mockPaypalService, VerificationModeFactory.times(times)).storeMetaData(
-      captor[Payment],
-      captor[Set[Allocation]],
-      captor[Option[String]],
-      captor[Option[String]],
-      captor[Option[String]],
-      captor[Option[String]],
-      captor[Option[String]],
-      captor[Option[String]],
-      captor[Option[IdentityId]],
-      captor[Option[String]],
-      captor[Option[String]]
-    )(captor[LoggingTags])
+    eventually(timeout(Span(60, Seconds)), interval(Span(1, Seconds))) {
+      def captor[A <: AnyRef](implicit classTag: ClassTag[A]): A =
```

Comment:
Personally I'd find it slightly easier to follow this if the captor function was declared outside of the eventually

Reply:
Hi Jacob, thats a good idea! 
Implementing now.